ui/src/buckets/components/DemoDataDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+// Libraries
+import React, {FC} from 'react'
+import _ from 'lodash'
+
+// Components
+import {IconFont, ComponentColor, Dropdown} from '@influxdata/clockface'
+
+// Types
+import {Bucket} from 'src/types'
+import {getDemoDataBucketMembership as getDemoDataBucketMembershipAction} from 'src/cloud/actions/demodata'
+
+interface Props {
+  buckets: Bucket[]
+  getMembership: typeof getDemoDataBucketMembershipAction
+}
+
+const DemoDataDropdown: FC<Props> = ({buckets, getMembership}) => {
+  const demoDataItems = buckets.map(b => (
+    <Dropdown.Item
+      testID={`dropdown-item ${b.name}`}
```

Comment:
do you want to prefix this testID with something like `demodata_dropdown-item` or something along those lines just to future proof?

Reply:
that makes sense